WordPress/src/main/res/values/strings.xml

```diff
@@ -151,6 +152,7 @@
     <string name="media_error_exceeds_memory_limit">File too large to be uploaded on this site</string>
     <string name="media_error_internal_server_error">Upload error. Try changing Optimize Images in your app\'s settings</string>
     <string name="media_error_timeout">Server took too long to respond</string>
+    <string name="media_error_incorrect_audio_format">We\’re sorry, this type of audio file is not supported</string>
```

Comment:
We should use the apostrophe character, like this: `We\'re sorry,[...]`

Reply:
Fixed.